gui/include/scanHelper.hh

```diff
@@ -0,0 +1,143 @@
+#ifndef EUDAQ_INCLUDED_GUI_ScanHelper
+#define EUDAQ_INCLUDED_GUI_ScanHelper
+
+#include <stdio.h>
+#include <string>
+#include <vector>
+#include "eudaq/Config.hh"
+#include "eudaq/Logger.hh"
+#include "eudaq/RunControl.hh"
+
+using namespace std;
```

Comment:
no default ```namespace std``` at header file.  It will affect all the other files which include this ScanHelper.hh

Reply:
true - remnant from testing